slack.ts

```diff
@@ -0,0 +1,69 @@
+import * as querystring from "querystring";
+import fetch from "node-fetch";
+
+const tkn = process.env.SLACK_TOKEN;
+const domain = process.env.SLACK_DOMAIN;
+
+interface Config {
+    channels: string[];
+}
+
+async function sendMessage(message : string, config : any) : Promise<void> {
+    for (var i = 0; i < config.channels.length; i++) {
```

Comment:
One of these `postMessage` API calls might fail, and its slower to run them one at a time when they can all be done concurrently. I would use `Promise.all` to run them all at the same time:

```typescript
await Promise.all(config.channels.map(channel => (async () => {
  try {
    await send(channel);
  }
  catch (err) {
    // ...
  }
})));
```